HiP-ThumbnailService/Utility/DirConfig.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace PaderbornUniversity.SILab.Hip.ThumbnailService.Utility
+{
+    public class DirConfig
```

Comment:
Please rename to `DirectoryConfig` for clarity (yeah `AuthConfig` is also not perfect, I know 😉)